manimo/sensors/real_sense_camera.py

```diff
@@ -49,17 +49,23 @@ def add_image(
             time.sleep(1.0 / camera_cfg.hz)
 
     align = rs.align(rs.stream.color)
+    depth_processing = (
+            rs.hole_filling_filter(),
+            rs.temporal_filter(),
+        )
```

Comment:
put this under a flag but make it default true since our code uses this

Reply:
but what's the point of disabling this? I haven't seen any negative impact of this depth processing on the point cloud yet...

Reply:
just cleaner to put it under a flag since its not standard to this code base. you can make the default true

Reply:
updated